pull.py

```diff
@@ -103,8 +146,41 @@
     'FDL1.2': {'spdx': 'GFDL-1.2'},
     'FDL1.3': {'spdx': 'GFDL-1.3'},
     'FreeBSD': {'spdx': 'BSD-2-Clause-FreeBSD'},
+    'freetype': {'spdx': 'FTL'},
     'GNUAllPermissive': {'spdx': 'FSFAP'},
     'GNUGPLv3': {'spdx': 'GPL-3.0'},
+    'gnuplot': {'spdx': 'gnuplot'},
+    'GPLv2': {'spdx': 'GPL-2.0'},
+    'HPND': {'spdx': 'HPND'},
+    'IBMPL': {'spdx': 'IPL-1.0'},
+    'iMatix': {'spdx': 'iMatix'},
+    'imlib': {'spdx': 'Imlib2'},
+    'ijg': {'spdx': 'IJG'},
+    'intel': {'spdx': 'Intel'},
+    'IPAFONT': {'spdx': 'IPA'},
+    'ISC': {'spdx': 'ISC'},
+    'JSON': {'spdx': 'JSON'},
+    'LAL-1.2': {'spdx':'LAL-1.2'},
+    'LAL-1.3': {'spdx':'LAL-1.3'},
+    'LGPLv3': {'spdx': 'LGPL-3.0'},
+    'LGPLv2.1': {'spdx': 'LGPL-2.1'},
+    'LPPL-1.2': {'spdx': 'LPPL-1.2'},
+    'LPPL-1.3a': {'spdx': 'LPPL-1.3a'},
+    'lucent102': {'spdx': 'LPL-1.02'},
+    'ModifiedBSD': {'spdx': 'BSD-3-Clause'},
+    'MPL': {'spdx': 'MPL-1.1'},
+    'MPL-2.0': {'spdx':'MPL-2.0'},
+    'ms-pl': {'spdx': 'MPL-1.1'},
```

Comment:
This is FSF's [Microsoft Public License (Ms-PL)][1], which is [SPDX's MS-PL][2].  [SPDX's MPL-1.1 is the Mozilla Public License 1.1][3].

[1]: https://www.gnu.org/licenses/license-list.html#ms-pl
[2]: https://github.com/spdx/license-list-XML/blob/e5da40e25becb0aa7626d3f62649d2387284a623/src/MS-PL.xml
[3]: https://github.com/spdx/license-list-XML/blob/e5da40e25becb0aa7626d3f62649d2387284a623/src/MPL-1.1.xml

Reply:
Oops - that must have been a typo.  Good catch.